synopsys-task/src/synopsys-task/model/polaris.ts

```diff
@@ -1,5 +1,7 @@
+import { Reports } from "./reports";
 export interface Polaris {
   polaris: PolarisData;
+  reports?: Reports;
```

Comment:
why we are adding in in polaris inputs, for this story we have to be focused on black duck